### PR TITLE
preventing jar files needs preventing zip files as well

### PR DESCRIPTION
**Pull request by zalun**

### CODE REVIEW

File: apps/jetpack/models.py [hunk 1]
```diff
@@ -62,6 +65,19 @@
 
 EDITABLE_EXTENSIONS = ("html", "css", "js", "txt", "xml", "json")
 
+def _raise_if_content_not_allowed(content):
+    fp = StringIO(content)
```
Comment (pennyfx) [inline apps/jetpack/models.py]: What does fp mean?  File Pointer?

File: apps/jetpack/errors.py [hunk 2]
```diff
@@ -19,6 +19,9 @@ class FilenameExistException(SimpleException):
 class IllegalFilenameException(SimpleException):
     " This filename contains illegal characters "
 
+class IllegalFileException(SimpleException):
+    " This fileis not allowed "
```
Comment (pennyfx) [inline apps/jetpack/errors.py]: typo   fileis -> file is

File: apps/jetpack/models.py (hunk 1 above)
Comment (zalun) [inline apps/jetpack/models.py]: Yes that's a copy&paste I should be more explicit

**Comment by pennyfx** [comment]
Looks ok.   r+

**Pull request merged**